cumulus_library/studies/core/core_templates/completion_utils.jinja

```diff
@@ -45,6 +45,7 @@ in this table.
                 AND BOOL_OR(ec.table_name = 'documentreference')
                 AND BOOL_OR(ec.table_name = 'medicationrequest')
                 AND BOOL_OR(ec.table_name = 'observation')
+                AND BOOL_OR(ec.table_name = 'procedure')
```

Comment:
So I think this deserves a tiny callout.

For context, the way completion works is that we ignore any encounters for which all the listed associated tables have not been loaded by the ETL yet.

So adding a new resource like this to the list of "required resources" means:
- _Transition pain_: if a site hasn't ETL'd procedures yet, the next time they rebuild `core`, all encounters will disappear.
- _Ongoing pain_: `core` requires more and more resources (like procedures) to be exported and ETL'd before it can function as expected

However... This is also... the point of completion? Would we want to draw a line between "resources we actually care about" and "resources we do technically support"? If we didn't add Procedure here, and your study used `core__procedure`, you would now be subject to the reasons we added the completion table - engineers ETL'ing data can mess you up.

This "ignore encounters" trick was so that studies didn't have to know about the whole completion table feature. They just would `inner join` on encounters at some point and incomplete data would be ignored. But maybe we have a list of 2nd-tier resources for which studies are expected to manually check the completion tables? (the logic isn't necessarily fun, but we could create some docs with examples)

Or we just accept the pains listed above as we grow.